### DISCUSSION TITLE
Nginx Proxy Manager Crashing

### DISCUSSION THREAD

**Discussion by tsely** [tteck/Proxmox#2878] [Help Needed]
Hello, I'm fairly new with Proxmox and everything in general. But I have encountered some weird issues with NPM, that I'm not sure is due to the script for installing it or something wrong with NPM itself.

1. NPM v2.11.1 would crash or fail to disable/enable proxy hosts that I have enabled or disabled once or twice.
2. Restarting the LXC causes NPM (v2.10.4 and v2.11.1) to not start back up.
3. Was also having Stream issues but not sure if that was due to NPM or my VM not accessing it properly.

The only way I was able to fix these two issues were to completely rerun the script to install NPM LXC all over again. It has been very frustrating as I've been having this issue for nearly a week now. I originally thought it was an issue with my Proxmox setup but everything else (Pi-Hole, Wireguard, Vaultwarden, and Pterodactyl) seems to be running fine. Any help would be amazing, thank you in advance!

**Comment by tteck** [comment]
I'm unsure why you're experiencing these issues, but rest assured, the Nginx Proxy Manager LXC script is functioning correctly without any issues on my end.
